webapp/components/Upload/index.vue

```diff
@@ -44,6 +45,14 @@ export default {
       }
     },
   },
+  watch: {
+    error() {
+      let that = this
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-03T11:17:12Z" title="Monday, June 3rd 2019, 1:17:12 pm +02:00">Jun 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -44,6 +45,14 @@ export default {
       }
     },
   },
+  watch: {
+    error() {
+      let that = this
```


Was this not solved with EcmaScript? :thinking: could you try out with `error: () => {` arrow functions?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-03T11:17:59Z" title="Monday, June 3rd 2019, 1:17:59 pm +02:00">Jun 3, 2019</time>_

---


Or, what is `this ` actually here? Could you explain? :thinking:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-04T00:04:58Z" title="Tuesday, June 4th 2019, 2:04:58 am +02:00">Jun 4, 2019</time>_

---


I tried it out and it seems to work, but it breaks the tests, so maybe it doesn't or maybe the tests need to be modified somehow... to be honest, we copied and pasted this from the `vue2-dropzone` demo...